src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -195,7 +195,7 @@ public Path getJavaAppEngineSdkPath() {
     return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
   }
 
-  private Path getJavaToolsJar() {
+  public Path getJavaToolsJar() {
```

Comment:
I think it could be fair that the client has some knowledge of the SDK structure. The alternative would be to have a method to return each and every one of the SDK JARs, which might be a bit unsightly.
What do you guys think?